src/components/sections/Incentives/Incentives.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import type { ReactNode } from 'react'
+import { Incentive as UIIncentive, List as UIList } from '@faststore/ui'
+
+import './incentives.scss'
+
+interface Incentive {
+  icon: ReactNode
+  title?: string
+  text: string
+}
+
+interface Props {
+  incentives: Incentive[]
+  hasBackground?: boolean
+}
+
+function Incentives({ incentives, hasBackground }: Props) {
+  return (
+    <div
+      className="incentives / grid-content-full"
+      data-incentives-background={hasBackground}
```

Comment:
If we pass `hasBackground={false}`, the background is applied yet. Maybe we can check something like:
```suggestion
      data-incentives-background={hasBackground || undefined}
```

after that the `eslint` is gonna suggest to use `nullish-coalescing (??) operator` instead of `logical (||) operator`, but the results are different. So I recommend disabling this rule in the `.eslint`

```
"rules": {
    "@typescript-eslint/prefer-nullish-coalescing": "off"
  }
```

I wonder if instead of a boolean flag we can pass either the `background-color` and styles inline or `variants` (class) with CSS ready. But it's not blocking, it's up to you hahah.

Reply:
hmm, maybe we can pass a class here. I'll try that! Thanks!